src/Microsoft.AspNetCore.Sockets.Http/Internal/Transports/WebSocketsTransport.cs

```diff
@@ -64,11 +64,11 @@ public WebSocketsTransport(WebSocketOptions options, IDuplexPipe application, De
             }
         }
 
-        public async Task ProcessSocketAsync(WebSocket socket)
+        public async Task ProcessSocketAsync(WebSocket socket, CancellationToken cancellationToken)
```

Comment:
Should this be public? It is only used by this class except for unit tests.

Reply:
It's because of the tests that it's public, since it's "pubternal". I'm fine with making it `InternalsVisibleTo` but @davidfowl might complain (not necessarily something to be avoided ;)).

Reply:
I also like `InternalsVisibleTo` over making things public for testing

Reply:
Yeah, generally if it's only for testing, we also lean to `InternalsVisibleTo`. Pubternal is supposed to be for those things that the super advanced user might actually want to tweak. Let's `IVT` it up.